pyaedt/hfss.py

```diff
@@ -4651,6 +4768,171 @@ def assign_radiation_boundary_to_faces(self, faces_id, boundary_name=""):
             rad_name = generate_unique_name("Rad_")
         return self.create_boundary(self.BoundaryType.Radiation, faces_list, rad_name)
 
+    @pyaedt_function_handler()
+    def assign_impedance_boundary_to_faces(
+        self, faces_id, boundary_name="", resistance=50, reactance=0, is_infground=False
+    ):
+        """Assign an impedance boundary to one or more faces.
+
+        Parameters
+        ----------
+        faces_id :
+            Face ID(s) to assign the impedance boundary condition to.
```

Comment:
```suggestion
            One or more face IDs to assign the impedance boundary condition to.
```
The data type needs to be added here and to a few other of the following parameters.